packages/web-client/app/components/card-pay/merchant-card/index.hbs

```diff
@@ -0,0 +1,29 @@
+<article
+  class="merchant-card"
+  style={{css-var merchant-logo=@logo}}
+  data-test-merchant-card-logo={{@logo}}
+  data-test-merchant-card={{@id}}
+>
```

Comment:
Add splattributes? We're definitely going to need a way to add a click handler, for example.